html/dom/dynamic-markup-insertion/document-write/reload.window.js

```diff
@@ -0,0 +1,35 @@
+async_test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe"));
+  let reloaded = false;
+  frame.src = "/common/blank.html";
+  frame.onload = t.step_func(() => {
+    if (reloaded) {
+        assert_equals(frame.contentDocument.body.textContent, "");
+        t.done();
+        return;
+    }
+    assert_false(reloaded);
+    assert_equals(frame.contentDocument.body.textContent, "");
+    frame.contentDocument.write("Hey");
+    assert_equals(frame.contentDocument.body.textContent, "Hey");
+    reloaded = true;
+    frame.contentWindow.location.reload();
+  });
+}, "document.write() and reloading");
```

Comment:
Yeah, I start out somewhat vague with all of these since I'm not quite sure where we want to end up yet, but I'm happy to clean them up before landing.